app/Models/Entry.php

```diff
@@ -185,7 +185,7 @@ public function content(bool $withEnclosures = true, bool $allowDuplicateEnclosu
 				continue;
 			}
 			$credit = $enclosure['credit'] ?? '';
-			$description = $enclosure['description'] ?? '';
+			$description = nl2br($enclosure['description'] ?? '');
```

Comment:
```suggestion
			$description = nl2br($enclosure['description'] ?? '', true);
```

Use XML-compatible mode when we can